lib/globus/client/endpoint.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "faraday"
+
+module Globus
+  class Client
+    # The namespace for the "login" command
+    class Endpoint
+      def initialize(token)
+        @token = token
+      end
+
+      attr_reader :token
+
+      def connection
+        # Transfer API connection
+        Faraday.new(
+          url: Settings.globus.transfer_url,
+          headers: { Authorization: "Bearer #{token}" }
+        )
+      end
+
+      def length
```

Comment:
Is this method temporary, for facilitating tests on the initial set-up? I'm not sure what "length" and "total" fields are in the response, and whether they're always the same--they're not documented so far as I've been able to find. So would be uneasy using them in the gem until we understand them better. If they're just here as we're getting the gem set up and confirming a test can run, OK to keep in for now. Maybe add a comment so we don't forget?

Reply:
Essentially temporary yes, mostly just something to test parsing the data, though it shows perhaps some useful information (I think the number of objects returned) that could be used for validation, etc.